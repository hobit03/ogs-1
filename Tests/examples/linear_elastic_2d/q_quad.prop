<?xml version="1.0"?>
<coupling>
	<P algorithm="Serial" convergence="FemFunctionConvergenceCheck" max_itr="1" epsilon="1e-4">
		<out>u</out>
		<out>gp_strain</out>
		<out>gp_stress</out>
		<out>nod_strain</out>
		<out>nod_stress</out>
		<problems>
			<M name="DEFORMATION">
				<out>u</out>
			</M>
			<M name="ELEMENT_STRESS_STRAIN">
				<in>u</in>
				<out>gp_strain</out>
				<out>gp_stress</out>
			</M>
			<M name="NODAL_STRESS_STRAIN">
				<in>gp_strain</in>
				<in>gp_stress</in>
				<out>nod_strain</out>
				<out>nod_stress</out>
			</M>
		</problems>
	</P>
</coupling>

