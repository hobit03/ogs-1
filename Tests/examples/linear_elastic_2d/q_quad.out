GeoSys-OUT: Output ------------------------------------------------
#OUTPUT
 $NOD_VALUES
  DISPLACEMENT
  STRAIN
  STRESS
  DISPLACEMENT_X1
  DISPLACEMENT_Y1
  STRESS_XX
  STRESS_XY
  STRESS_YY
  STRESS_ZZ
  STRESS_XZ
  STRESS_YZ
  STRAIN_XX
  STRAIN_XY
  STRAIN_YY
 $GEO_TYPE
  DOMAIN
 $DAT_TYPE
  PVD
 $TIM_TYPE
  STEPS 1
#STOP


